esp/esp/themes/theme_data/fruitsalad/templates/main.html

```diff
@@ -209,17 +209,29 @@ <h1 class="title">{% block content_title %}&nbsp;{% endblock %}</h1>
 {% endblock %}
 
       <div id="content">
+          {% if request.show_perm_info %}
+          <div class="alert alert-info">
+              <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+              Permission/deadline controlling this page: {{ request.perm_names|join:", " }}
```

Comment:
super nit: I'd say "Permission/deadline required to view this page" -- and you may as well link to the manage deadlines page, iff that's not too hard to do.